core/src/test/scala/integration/kafka/api/ParallelControllerStartupTest.scala

```diff
@@ -0,0 +1,122 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information regarding copyright
+ * ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package kafka.api
+
+import scala.collection.JavaConverters._
+
+import java.util.Properties
+
+import org.junit.jupiter.api.{BeforeEach, Test}
+import org.junit.jupiter.api.Assertions._
+
+import kafka.server.KafkaConfig
+import kafka.utils.TestUtils
+import org.apache.kafka.clients.admin._
+
+/**
+ * Scalability/performance test for sequential vs. parallel controller startup, at least when run
+ * manually with maxTopics set to 1500.  KafkaController doesn't expose its detailed timings outside
+ * of logs, however, so the test case doesn't explicitly <em>verify</em> parallel startup, though it
+ * does configure it explicitly.
+ */
+class ParallelControllerStartupTest extends AbstractConsumerTest {
```

Comment:
Since we are not using the producer or consumer in this class, can we make it inherit from the IntegrationTestHarness?